packit_service/worker/handlers/abstract.py

```diff
@@ -237,16 +240,34 @@ def get_tag_info(self) -> dict:
             tags.update({"package_name": self.data.event_dict["package_name"]})
         return tags
 
+    def log_memory_stats(self) -> None:
+        gc.collect()
+        memory_usage = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss
+        logger.error(f"Memory usage: {memory_usage} (kb)")
```

Comment:
do we need this as error level? It will end up in Sentry iirc

Reply:
No, I forgot to change it...